.prettierrc

```
@@ -2,6 +2,9 @@
   "trailingComma": "all",
   "tabWidth": 2,
   "semi": true,
-  "singleQuote": true
+  "singleQuote": true,
+  "importOrder": ["^@?fastify*", "^@?graasp*", "^[./]"],
```

I just noticed I initially wrote the regex wrong in `graasp-template-backend` 🤦 It works though, but the `*` token is useless. It should either be

```suggestion
  "importOrder": ["^@?fastify(.*)", "^@?graasp(.*)", "^[./](.*)"],
```

or
```suggestion
  "importOrder": ["^@?fastify", "^@?graasp", "^[./]"],
```

I think either works. Anyway I'll have to change it everywhere 💀